changelogs/unreleased/gh-8861-flightrec-options.md

```diff
@@ -0,0 +1,3 @@
+## feature/config
```

Comment:
Needs rebase after PR #8877 and update of the ignore list in the corresponding test case.

Reply:
Done.